doc/api_assets/style.css

```diff
@@ -338,7 +338,7 @@ hr {
   margin-bottom: 0;
 }
 
-p tt, p code, li code {
+p tt, p code, li code, h2 code, h3 code, h4 code, h5 code {
```

Comment:
Care to investigate if we can reduce this selector to `tt, code`? It's a bit hackish having to specify all the different ancestors.